src/plugins/intel_cpu/src/emitters/tpp/aarch64/kernel_executors/brgemm.cpp

```diff
@@ -0,0 +1,138 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "brgemm.hpp"
+
+#include "emitters/tpp/common/utils.hpp"
+#include "transformations/tpp/common/op/brgemm.hpp"
+
+using namespace Xbyak;
+using namespace dnnl::impl;
+
+namespace ov {
+namespace intel_cpu {
+namespace aarch64 {
+#define COMPILE_BRGEMM_TPP_KERNEL(...)                                        \
+    [&]() {                                                                   \
+        setenv("LIBXSMM_X86_HINT_USE_HIGH_PREC_ELTWISE_APPROX", "1", 1);      \
+        setenv("LIBXSMM_GEMM_K_A_PF_DIST", "4", 1);                           \
+        auto res = reinterpret_cast<const libxsmm_gemmfunction>(__VA_ARGS__); \
+        unsetenv("LIBXSMM_X86_HINT_USE_HIGH_PREC_ELTWISE_APPROX");            \
+        unsetenv("LIBXSMM_GEMM_K_A_PF_DIST");                                 \
+        return res;                                                           \
+    }()
+
+BrgemmKernelConfig::BrgemmKernelConfig(const element::Type& in0_dtype,
+                                       const element::Type& in1_dtype,
+                                       dnnl::impl::cpu::aarch64::cpu_isa_t primitive_isa)
+    : BrgemmBaseKernelConfig(),
+      m_static_params(std::make_shared<StaticParams>(in0_dtype, in1_dtype, primitive_isa)) {
+    m_hash = compute_hash();
+}
+
+BrgemmKernelConfig::StaticParams::StaticParams(const element::Type& in0_dtype,
+                                               const element::Type& in1_dtype,
+                                               dnnl::impl::cpu::aarch64::cpu_isa_t primitive_isa)
+    : StaticBaseParams(in0_dtype, in1_dtype, dnnl::impl::cpu::x64::cpu_isa_t::isa_undef, compute_hash(primitive_isa)) {
+    m_type_in0 = tpp::ov_to_xsmm_dtype(in0_dtype);
+    m_type_in1 = tpp::ov_to_xsmm_dtype(in1_dtype);
+    m_type_exec = LIBXSMM_DATATYPE_F32;
+    m_type_out0 = LIBXSMM_DATATYPE_F32;
+    m_compile_flags = LIBXSMM_GEMM_FLAGS('N', 'N');
+    m_prefetching_flags = false;
+    isa = primitive_isa;
+}
+
+size_t BrgemmKernelConfig::StaticParams::compute_hash(dnnl::impl::cpu::aarch64::cpu_isa_t aarch_isa) {
+    return hash_combine(0, aarch_isa);
+}
```

Comment:
We don't use `aarch_isa` in this config/kernel/executor. Then this attribute should be missed in params

Reply:
`aarch_isa` is removed, Thanks Alexandra!